dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/fieldsmapper/TrackedEntityFieldsParamMapperTest.java

```diff
@@ -58,46 +65,20 @@ void getTrackedEntityInstanceParamsWithStar()
     }
 
     @Test
-    void getTrackedEntityInstanceParamsWithPresetAll()
+    void mapWithOnlyRelationships()
     {
-
-        TrackedEntityInstanceParams params = map( List.of( ":all" ) );
+        TrackedEntityInstanceParams params = map( "relationships" );
```

Comment:
didn't we agree on the shouldXXXWhenYYY standard for the naming convention?

Reply:
After the last backend meeting it was not so clear anymore 🤔 since this is 2.38 these tests will reach their end of life at some point. I can have a look at the ones in master.